content-services/content-sources-backend
# Fixes 4370: template create API allows blank arch/version

## Summary

Checks for arch/version before template creation and rejects request if either is missing or blank

## Testing steps

- Try to create a template with no arch or version 
- Request should be rejected and return a 400

## Checklist
